build.gradle.kts

```diff
@@ -17,8 +17,10 @@ repositories {
 
 buildscript {
     dependencies {
-        classpath("software.amazon.smithy:smithy-openapi:1.21.0")
-        classpath("software.amazon.smithy:smithy-aws-traits:1.21.0")
+        val smithyVersion = "1.21.0"
```

Comment:
Is there a convention where these version declarations go, like a .properties file?